logstash-core/lib/logstash/api/commands/node.rb

```diff
@@ -62,7 +62,7 @@ def initialize(cmd, options)
           end
           
           def to_s
-            hash = to_hash
+            hash = to_hash[:hot_threads]
```

Comment:
Isn't this still JSON output? I'm fine with this change, but I wonder if we need another issue to write true human readable hot_threads output.

If I hit `curl 'http://localhost:9600/_node/hot_threads?h'` on this branch I still get JSON.